src/gui/C4Gui.h

```diff
@@ -1284,10 +1284,10 @@ namespace C4GUI
 
 	private:
 		void Deselect(); // clear selection range
-		void ClearText();    // remove all the text
 
 	public:
 		bool InsertText(const char *szText, bool fUser); // insert text at cursor pos (returns whether all text could be inserted)
+		void ClearText();    // remove all the text
```

Comment:
Not sure about that.. maybe "friend" the class that uses ClearText() and leave it private?

Reply:
It's fine this way. I don't understand why ClearText was private in the first place.

Reply:
I think visibility in the C4Gui stuff is strictly by actual use and not to make a sane interface...